src/tasks/__tests__/processBulkUploadTask.int.test.ts

```diff
@@ -139,22 +106,28 @@ describe('processBulkUploadTask', () => {
 			systemUserAuthContext,
 			proposalsDataFile.id,
 		);
-		const requests = await mockS3ResponsesForBulkUploadTaskProcessing(
-			bulkUploadTask,
-			path.join(
-				__dirname,
-				'fixtures',
-				'processBulkUploadTask',
-				'validCsvTemplate.csv',
+
+		s3Mock.on(GetObjectCommand).resolves({
+			Body: sdkStreamMixin(
+				fs.createReadStream(
+					path.join(
+						__dirname,
+						'fixtures',
+						'processBulkUploadTask',
+						'validCsvTemplate.csv',
+					),
+				),
 			),
-		);
+		});
+
 		await processBulkUploadTask(
 			{
 				bulkUploadId: bulkUploadTask.id,
 			},
 			getMockJobHelpers(),
 		);
-		expect(requests.getRequest.isDone()).toEqual(true);
+
+		expect(s3Mock.commandCalls(GetObjectCommand).length).toBeGreaterThan(0);
```

Comment:
I think this can be more specific here (or rather, I think this can be `.toEqual(1)`